bindings/python/rgbmatrix/core.pyx

```diff
@@ -53,6 +53,12 @@ cdef class Canvas:
                 b = (pixel >> 16) & 0xFF
                 my_canvas.SetPixel(xstart+col, ystart+row, r, g, b)
 
+cdef class FrameData:
+    cdef const char *__getData(self) except +:
```

Comment:
```suggestion
    cdef const char *__getData(self) except NULL:
```
`except +` means that code can throw C++ exception
`except NULL` - check exception state if returned value is NULL